gazebo_plugins/include/gazebo_plugins/gazebo_ros_openni_kinect.h

```diff
@@ -115,7 +115,10 @@ namespace gazebo
     private: sensor_msgs::PointCloud2 point_cloud_msg_;
     private: sensor_msgs::Image depth_image_msg_;
 
+    /// \brief Minimum range of the point cloud
     private: double point_cloud_cutoff_;
```

Comment:
I certainly could -- I was trying to do the minimum set of changes. As for renaming the parameter in SDF -- thats pretty problematic for current users.